### PR TITLE
#1802 Update NniOnWindows.md

### PR DESCRIPTION
**Pull request by scarlett2018**
fix problem raised in #1802

### CODE REVIEW

File: docs/en_US/Tutorial/NniOnWindows.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
-# NNI on Windows (experimental feature)
+# NNI on Windows
+As Linux version, NNI on Windows supports local, remote as well as cloud integration mode for OpenPAI and other training sercies.
```
Comment (xuehui1991) [inline docs/en_US/Tutorial/NniOnWindows.md]: 我觉得这里有个很困惑的点是， 假如我手里有几台windows的机器，我能用一台去启动剩下的remote模式的nni吗？（我觉得有必要在这里指出这一点）

Comment (scarlett2018) [reply docs/en_US/Tutorial/NniOnWindows.md]: 明白，等倪总confirm一下@leckie-chn。。。

Comment (chicm-ms) [reply docs/en_US/Tutorial/NniOnWindows.md]: @xuehui1991 目前remote machine training service  支持在Windows上跑nni manager去连接remote linux （在linux上跑trial）,   但是不支持Windows 被nni manager连接.    要想在windows系统上跑trial job，只能用local training service.

Comment (QuanluZhang) [reply docs/en_US/Tutorial/NniOnWindows.md]: @leckie-chn

Comment (leckie-chn) [reply docs/en_US/Tutorial/NniOnWindows.md]: @xuehui1991 NNI windows只支持local模式，换句话说不能把windows机器当作remote的training service :)

Comment (xuehui1991) [reply docs/en_US/Tutorial/NniOnWindows.md]: @scarlett2018  Seems we need to change here.

Comment (scarlett2018) [reply docs/en_US/Tutorial/NniOnWindows.md]: > @scarlett2018 Seems we need to change here.

Updated

File: docs/en_US/Tutorial/NniOnWindows.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
-# NNI on Windows (experimental feature)
+# NNI on Windows
+Differ from Linux version, NNI only supports local mode right now. Please comment in [NNI's backlog issue](https://github.com/microsoft/nni/issues/1917) if you have needs on other modes for NNI on Windows.
```
Comment (QuanluZhang) [inline docs/en_US/Tutorial/NniOnWindows.md]: ... right now. For other modes, you could use Windows as a machine to start an NNI experiment. For example, in remote mode, you could use Windows to run `nnictl` commands and it connects several Linux machines for running trials. Similarly, you could start an NNI experiment on Windows to run trials in PAI mode. But note that Windows cannot act as a remote server for running trials.

Comment (scarlett2018) [reply docs/en_US/Tutorial/NniOnWindows.md]: > But note that Windows cannot act as a remote server for running trials.
we might need one diagram to explain nni's runtime nodes, like: nnictl node + trials node(s).

Comment (QuanluZhang) [reply docs/en_US/Tutorial/NniOnWindows.md]: agree

Comment (scarlett2018) [reply docs/en_US/Tutorial/NniOnWindows.md]: Updated, @xuehui1991, @leckie-chn , @QuanluZhang - please help to review.

File: docs/en_US/Tutorial/Installation.md [hunk 3]
```diff
@@ -1,6 +1,18 @@
-# Installation of NNI
+# Plan your topology
```
Comment (squirrelsc) [inline docs/en_US/Tutorial/Installation.md]: The topology concept makes end users feeling complex, and NNI is not so complex actually. I suggest not to add this section. If people want to implement new trainning platform, they need to know this. So it's more like design related things, shouldn't be put in installation part.

Comment (QuanluZhang) [reply docs/en_US/Tutorial/Installation.md]: agree with @squirrelsc

Comment (scarlett2018) [reply docs/en_US/Tutorial/Installation.md]: @squirrelsc  @QuanluZhang - agree the concern. while still think we need to have this document in a lower exposure place. If you agree this, where would you like to recommend for putting it?

**Comment by scarlett2018** [comment]
As discussed, @squirrelsc will refactor the remote md instead of making this change.

**Pull request closed without merging**